test/e2e/tests/metamask-responsive-ui.spec.js

```diff
@@ -0,0 +1,233 @@
+const { withFixtures, tinyDelayMs } = require('../helpers');
+const enLocaleMessages = require('../../../app/_locales/en/messages.json');
+
+describe('Metamask Responsive UI', function () {
+  it('Creating a new wallet', async function () {
+    const driverOptions = { responsive: true };
+
+    await withFixtures(
+      {
+        fixtures: 'onboarding',
+        driverOptions,
+        title: this.test.title,
+        failOnConsoleError: false,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+
+        // clicks the continue button on the welcome screen
+        await driver.findElement('.welcome-page__header');
+        await driver.clickElement({
+          text: enLocaleMessages.getStarted.message,
+          tag: 'button',
+        });
+        await driver.delay(tinyDelayMs);
+
+        // clicks the "Create New Wallet" option
+        await driver.clickElement({ text: 'Create a Wallet', tag: 'button' });
+
+        // clicks the "I Agree" option on the metametrics opt-in screen
+        await driver.clickElement('.btn-primary');
+
+        // accepts a secure password
+        await driver.fill(
+          '.first-time-flow__form #create-password',
+          'correct horse battery staple',
+        );
+        await driver.fill(
+          '.first-time-flow__form #confirm-password',
+          'correct horse battery staple',
+        );
+        await driver.clickElement('.first-time-flow__checkbox');
+        await driver.clickElement('.first-time-flow__form button');
+
+        // reveals the seed phrase
+        await driver.clickElement(
+          '.reveal-seed-phrase__secret-blocker .reveal-seed-phrase__reveal-button',
+        );
+        const revealedSeedPhrase = await driver.findElement(
+          '.reveal-seed-phrase__secret-words',
+        );
+        const seedPhrase = await revealedSeedPhrase.getText();
+        assert.equal(seedPhrase.split(' ').length, 12);
+
+        await driver.clickElement({
+          text: enLocaleMessages.next.message,
+          tag: 'button',
+        });
+
+        async function clickWordAndWait(word) {
+          await driver.clickElement(
+            `[data-testid="seed-phrase-sorted"] [data-testid="draggable-seed-${word}"]`,
+          );
+          await driver.delay(tinyDelayMs);
+        }
+
+        // can retype the seed phrase
+        const words = seedPhrase.split(' ');
+        for (const word of words) {
+          await clickWordAndWait(word);
+        }
+        await driver.clickElement({ text: 'Confirm', tag: 'button' });
+
+        // clicks through the success screen
+        await driver.findElement({ text: 'Congratulations', tag: 'div' });
+        await driver.clickElement({
+          text: enLocaleMessages.endOfFlowMessage10.message,
+          tag: 'button',
+        });
+
+        // Show account information
+        // show account details dropdown menu
+        await driver.clickElement(
```

Comment:
Nit: I'm not sure how useful this part is, or the lock step at the end :thinking:  I guess we need to do _something_ after the account is created, to verify that it worked. Checking the balance would suffice though - that's what we do in the other flow.

Reply:
I had the same thought while working on it, then decided to update it based on the review comments 😅